### DISCUSSION TITLE
Trouble with remote

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#302] [General]
I was forced to reset my slingbox because I failed to get the admin password before the server shutdown. The slingbox came online and attached to the router with an IP and port (5201 instead of the previous 5216). I can stream video via vlc with the python code running on a Pi3B. However, the remote is not working. I have a FIOS STB and have loaded the FIOS_remote.txt in the config.ini.

I can bring up the remote page via a browser and according to the log it's sending codes, but nothing seems to happen.

Could it be the reset has screwed up the remote on the slingbox?

**Comment by GerryDazoo** [comment]
When you did the reset it wiped out all the remote IR configuration in the Slingbox. You'll see in the logs that Keycodes=[]
People are working on it.

**Reply by socelillyhammer** [reply]
I guess we are sitting tight to future remote codes that should solve the inactive remote problem with FIOS and other boxes?
